src/contracts/core/DelegationManager.sol

```diff
@@ -513,6 +769,57 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         return _operatorDetails[operator].stakerOptOutWindowBlocks;
     }
 
+    /**
+     * @notice Returns the number of actively-delegatable shares a staker has across all strategies
+     */
+    function getDelegatableShares(address staker) public view returns (IStrategy[] memory, uint256[] memory) {
+        // Get currently active shares and strategies for `staker`
+        int256 podShares = eigenPodManager.podOwnerShares(staker);
+        (IStrategy[] memory strategyManagerStrats, uint256[] memory strategyManagerShares) 
+            = strategyManager.getDeposits(staker);
+
+        // Has shares in StrategyManager, but not in EigenPodManager
+        if (podShares <= 0) {
+            return (strategyManagerStrats, strategyManagerShares);
+        }
```

Comment:
Maybe clarify comment - this branch might also mean the staker has no shares anywhere. If so, this is returning empty arrays.

Reply:
addressed here https://github.com/Layr-Labs/eigenlayer-contracts/pull/222/commits/08a3c0642b959ba1c56e401fb0048572acaa9469